.env

```diff
@@ -1,7 +1,2 @@
-VEGA_SIM_VEGA_TAG=dfad7923af512fac5f0b53f4552cf8e5b2f3bf65
```

Comment:
@TomMcL I think the update script removed some important environment variables again.